index.js

```diff
@@ -53,92 +54,106 @@ const subscribeOnConnectionInfoUpdates = (callback) => subscribeOnEvent(CONNECTI
-        status === undefined ? resolve() : reject(getError(status));
-    })
-});
-
-const createGroup = () => new Promise((resolve, reject) => {
-    WiFiP2PManager.createGroup(reasonCode => {
-        reasonCode === undefined ? resolve() : reject(getError(reasonCode));
-    })
-});
-
-const removeGroup = () => new Promise((resolve, reject) => {
-    WiFiP2PManager.removeGroup(reasonCode => {
-        reasonCode === undefined ? resolve() : reject(getError(reasonCode));
-    })
-});
+const connectWithConfig = (args) =>
+  new Promise((resolve, reject) => {
+    WiFiP2PManager.connectWithConfig(args, (status) => {
+      status === undefined ? resolve() : reject(getError(status));
+    });
+  });
+
+const cancelConnect = () =>
+  new Promise((resolve, reject) => {
+    WiFiP2PManager.cancelConnect((status) => {
+      status === undefined ? resolve() : reject(getError(status));
+    });
+  });
+
+const createGroup = () =>
+  new Promise((resolve, reject) => {
+    WiFiP2PManager.createGroup((reasonCode) => {
+      reasonCode === undefined ? resolve() : reject(getError(reasonCode));
+    });
+  });
+
+const removeGroup = () =>
+  new Promise((resolve, reject) => {
+    WiFiP2PManager.removeGroup((reasonCode) => {
+      reasonCode === undefined ? resolve() : reject(getError(reasonCode));
+    });
+  });
 
 const getAvailablePeers = () => WiFiP2PManager.getAvailablePeersList();
 
-const stopDiscoveringPeers = () => new Promise((resolve, reject) => {
-    WiFiP2PManager.stopPeerDiscovery(reasonCode => {
-        reasonCode === undefined ? resolve() : reject(getError(reasonCode));
-    })
-});
+const stopDiscoveringPeers = () =>
+  new Promise((resolve, reject) => {
+    WiFiP2PManager.stopPeerDiscovery((reasonCode) => {
+      reasonCode === undefined ? resolve() : reject(getError(reasonCode));
+    });
+  });
 
 const sendFile = (pathToFile) => WiFiP2PManager.sendFile(pathToFile);
 
-const receiveFile = (folder, fileName, forceToScanGallery = false) => new Promise((resolve, reject) => {
+const sendFileTo = (pathToFile, address) => WiFiP2PManager.sendFileTo(pathToFile, address);
+
+const receiveFile = (folder, fileName, forceToScanGallery = false) =>
+  new Promise((resolve, reject) => {
     WiFiP2PManager.receiveFile(folder, fileName, forceToScanGallery, (pathToFile) => {
-        resolve(pathToFile);
+      resolve(pathToFile);
     });
-});
+  });
 
 const sendMessage = (message) => WiFiP2PManager.sendMessage(message);
 
-const receiveMessage = () => new Promise((resolve, reject) => {
+const sendMessageTo = (message, address) => WiFiP2PManager.sendMessageTo(message, address);
+
+const receiveMessage = () =>
+  new Promise((resolve, reject) => {
     WiFiP2PManager.receiveMessage((message) => {
-        resolve(message);
+      resolve(message);
     });
-});
+  });
 
 const getConnectionInfo = () => WiFiP2PManager.getConnectionInfo();
 
 const getGroupInfo = () => WiFiP2PManager.getGroupInfo();
 
 export {
-    // public methods
```

Comment:
Could you also revert prettier changes in this file?

Reply:
Didn't see ^^' I've changed :)